app/controllers/comparisons/electricity_peak_kw_per_pupil_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Comparisons
+  class ElectricityPeakKwPerPupilController < BaseController
+    private
+
+    def title_key
```

Comment:
With the changes Deb has made to integrate the report, you'll need to define the report_key method, then you can remove the _introduction.html.erb file as the content will be loaded from the database.

Spec will also need the report created.